test/unit/services/azurerediscache/provision-spec.js

```diff
@@ -60,6 +60,27 @@ describe('RedisCache - Provision - PreConditions', function() {
     });
 });
 
+describe('RedisCache - Provision - PreConditions incorrect', function() {
+    var validParams = {};
+    var cp;
+        
+    before(function() {
+        validParams = {  /* missing parameters!! */
+            instance_id : 'b259c5e0-7442-46bc-970c-9912613077dd',            
+            azure : azure,
+            provisioning_result: '{\"provisioningState\":\"Creating\"}'
```

Comment:
I think, there should not be `provisioning_result` inside `validParams`.
It should be something like:
```
parameters: {
  location: 'southcentralus'
}
```
Other parameters are missing except `location`.